protocol.go

```diff
@@ -62,6 +63,7 @@ type Conn struct {
 	useConnAddr        bool
 	once               sync.Once
 	proxyHeaderTimeout time.Duration
+	UnknownOK          bool
```

Comment:
I don't believe this needs to be exported

Reply:
good catch. Un-exported.